### ISSUE TITLE
More letters busting right margin

### ISSUE THREAD

**Issue by visionario2** [notofonts/nastaliq#72]
version 4

I've noticed that some letters with dots like beh teh peh and yeh occurring at the beginning of lines break the right margin a little:

![inital beh teh yeh](https://github.com/user-attachments/assets/8c49167b-9830-4268-9099-fb76cbb283f8)
![initial peh](https://github.com/user-attachments/assets/84b02257-5733-45cb-bdfd-1dc130be4ba6)

Also final kaf can break the margin where there is only a single preceeding letter:

![final kaf](https://github.com/user-attachments/assets/1bb00d68-e974-4585-bd9c-3b6d0f526019)

**Comment by simoncozens** [comment]
If we fix this, the rasm doesn't line up against the margin... I'll check some printed sources.

**Comment by simoncozens** [comment]
Well, I checked a few sources. In many of them, particularly the hand-drawn lithographic books, there is no distinct "right margin"; the lines are pretty ragged. But in the MCLI edition of Mir Taqi Mir, which I consider to be the gold standard of Urdu typography, there is a distinct right margin and both initial dots and kaf strokes stay within it.

<img width="163" alt="Screenshot 2024-09-04 at 11 52 33" src="https://github.com/user-attachments/assets/51e30138-2dc1-4261-9fa2-356de272f2e3">

So yeah, I should probably try to fix this.

**Comment by visionario2** [comment]
Right, thanks. Also, if you look at some text set in Inpage (a widely used DTP package in Pakistan), the alignment to the right margin is consistent:
![inpage margin](https://github.com/user-attachments/assets/7bc85dee-a4d8-4861-8126-debae207153b)
